app/models/early_career_payments/eligibility.rb

```diff
@@ -1,5 +1,128 @@
+        claim_academic_year: AcademicYear.new(2022),
+        base_amount: 2_000,
+        uplift_amount: 3_000
+      ),
+      AwardAmount.new(
+        itt_subject: :chemistry,
+        itt_academic_year: AcademicYear.new(2020),
+        claim_academic_year: AcademicYear.new(2022),
+        base_amount: 2_000,
+        uplift_amount: 3_000
+      ),
+      AwardAmount.new(
+        itt_subject: :foreign_languages,
+        itt_academic_year: AcademicYear.new(2020),
+        claim_academic_year: AcademicYear.new(2022),
+        base_amount: 2_000,
+        uplift_amount: 3_000
+      ),
+      AwardAmount.new(
+        itt_subject: :mathematics,
+        itt_academic_year: AcademicYear.new(2018),
+        claim_academic_year: AcademicYear.new(2023),
+        base_amount: 5_000,
+        uplift_amount: 7_500
+      ),
+      AwardAmount.new(
+        itt_subject: :mathematics,
+        itt_academic_year: AcademicYear.new(2020),
+        claim_academic_year: AcademicYear.new(2023),
+        base_amount: 2_000,
+        uplift_amount: 3_000
+      ),
+      AwardAmount.new(
+        itt_subject: :physics,
+        itt_academic_year: AcademicYear.new(2020),
+        claim_academic_year: AcademicYear.new(2023),
+        base_amount: 2_000,
+        uplift_amount: 3_000
+      ),
+      AwardAmount.new(
+        itt_subject: :chemistry,
+        itt_academic_year: AcademicYear.new(2020),
+        claim_academic_year: AcademicYear.new(2023),
+        base_amount: 2_000,
+        uplift_amount: 3_000
+      ),
+      AwardAmount.new(
+        itt_subject: :foreign_languages,
+        itt_academic_year: AcademicYear.new(2020),
+        claim_academic_year: AcademicYear.new(2023),
+        base_amount: 2_000,
+        uplift_amount: 3_000
+      ),
+      AwardAmount.new(
+        itt_subject: :mathematics,
+        itt_academic_year: AcademicYear.new(2019),
+        claim_academic_year: AcademicYear.new(2024),
+        base_amount: 5_000,
+        uplift_amount: 7_500
+      ),
+      AwardAmount.new(
+        itt_subject: :mathematics,
+        itt_academic_year: AcademicYear.new(2020),
+        claim_academic_year: AcademicYear.new(2024),
+        base_amount: 2_000,
+        uplift_amount: 3_000
+      ),
+      AwardAmount.new(
+        itt_subject: :physics,
+        itt_academic_year: AcademicYear.new(2020),
+        claim_academic_year: AcademicYear.new(2024),
+        base_amount: 2_000,
+        uplift_amount: 3_000
+      ),
+      AwardAmount.new(
+        itt_subject: :chemistry,
+        itt_academic_year: AcademicYear.new(2020),
+        claim_academic_year: AcademicYear.new(2024),
+        base_amount: 2_000,
+        uplift_amount: 3_000
+      ),
+      AwardAmount.new(
+        itt_subject: :foreign_languages,
+        itt_academic_year: AcademicYear.new(2020),
+        claim_academic_year: AcademicYear.new(2024),
+        base_amount: 2_000,
+        uplift_amount: 3_000
+      )
+    ].freeze
```

Comment:
@nomasprime Any reason for not wanting to extact AwardAmounts into its own class? Other than the size it feels like its a bit of smell sitting it the policy::Eligibility class.
Happy to convinced that it should stay.

If not a fully fledged class then maybe a concern?

Reply:
It's just a basic struct, similar to a hash, normally written inline.

Wouldn't be in its own class for similar reasons why you wouldn't put a hash in its own class.

Definitely not a concern.